ArduCopter/Log.cpp

```diff
@@ -71,7 +71,12 @@ void Copter::Log_Write_Attitude()
     targets.z = wrap_360_cd(targets.z);
     ahrs.Write_Attitude(targets);
     ahrs_view->Write_Rate(*motors, *attitude_control, *pos_control);
-    if (should_log(MASK_LOG_PID)) {
+ }
+
+// Write PIDS packets
+void Copter::Log_Write_PIDS()
+{
+   if (should_log(MASK_LOG_PID)) {
```

Comment:
These functions usually unconditionally log, don't they?

Reply:
no....existing behaviour is that PIDs do not get logged without the bit , but they also dont get logged unless ATT is being logged, whether or not the PID mask bit is set